### PR TITLE
[port_utils] Add port alias mapping for z9332f

### PR DESCRIPTION
**Pull request by daall**
Signed-off-by: Danny Allen <daall@microsoft.com>

### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [x] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

#### What is the motivation for this PR?
To allow users to create testbeds with the 9332.

#### How did you do it?
Updated the port alias mapping to match the port_config.ini specified in sonic-buildimage.

#### How did you verify/test it?
Ran locally, confirmed output matched sonic-buildimage.

### CODE REVIEW

File: ansible/module_utils/port_utils.py [hunk 1]
```diff
@@ -112,4 +138,3 @@ def get_port_alias_to_name_map(hwsku):
             port_alias_to_name_map["Ethernet%d" % i] = "Ethernet%d" % i
 
     return port_alias_to_name_map
-
```
Comment (vaibhavhd) [inline ansible/module_utils/port_utils.py:115]: Line break at the eof removed.

**Review by vaibhavhd — APPROVED** [review]

**Comment by vaibhavhd** [comment]
Are the add-topo failures for this HWSKU resolved with this change?

**Review by neethajohn — APPROVED** [review]

File: ansible/module_utils/port_utils.py (hunk 1 above)
Comment (daall) [reply ansible/module_utils/port_utils.py:115]: Line break is still there, this was an extra newline. :)

**Comment by daall** [comment]
> Are the add-topo failures for this HWSKU resolved with this change?

Yes!

**Pull request merged**